go/tunnels/manager.go

```diff
@@ -56,8 +56,9 @@ var (
 var (
 	manageAccessTokenScope       = []TunnelAccessScope{TunnelAccessScopeManage}
 	hostAccessTokenScope         = []TunnelAccessScope{TunnelAccessScopeHost}
-	hostOrManageAccessTokenScope = []TunnelAccessScope{TunnelAccessScopeManage, TunnelAccessScopeHost}
-	readAccessTokenScope         = []TunnelAccessScope{TunnelAccessScopeManage, TunnelAccessScopeHost, TunnelAccessScopeConnect}
+	managePortsAccessTokenScopes = []TunnelAccessScope{TunnelAccessScopeManage, TunnelAccessScopeManagePorts, TunnelAccessScopeHost}
+	readAccessTokenScopes        = []TunnelAccessScope{TunnelAccessScopeManage, TunnelAccessScopeHost, TunnelAccessScopeConnect}
+	readPortsAccessTokenScopes   = []TunnelAccessScope{TunnelAccessScopeManage, TunnelAccessScopeManagePorts, TunnelAccessScopeHost, TunnelAccessScopeConnect}
```

Comment:
Not sure if the indentation is off on this line or if the diff is just displaying it that way.

Reply:
It looks correct to me. Try toggling the ignore-whitespace option on the diff view.

Reply:
Looks like it was actually the "split view" that was throwing me off :)